sampling/hierarchical/lib/opentelemetry/sampling/hierarchical/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
```

Comment:
I think the package the package naming should be symmetric with the SDK:

```ruby
OpenTelemetry::SDK::Trace::Samplers
````

Or more use the API namespace

```ruby
OpenTelemetry::Trace::Samplers
````

@open-telemetry/ruby-contrib-maintainers WDYT?